src/pymedphys/docker/orthanc/orthanc-and-miniconda/Dockerfile

```diff
@@ -19,4 +19,16 @@ RUN wget --quiet https://repo.anaconda.com/miniconda/Miniconda3-4.5.11-Linux-x86
 
 RUN conda update -n base -c defaults conda
 RUN conda config --add channels conda-forge
-RUN conda install pymedphys
\ No newline at end of file
+RUN conda install pymedphys && \
+  conda clean -tipsy
+
+RUN curl -sS https://deb.nodesource.com/gpgkey/nodesource.gpg.key | apt-key add -
+RUN echo 'deb https://deb.nodesource.com/node_10.x xenial main' | tee /etc/apt/sources.list.d/nodesource.list
+
+RUN curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add -
+RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+
+RUN apt-get update --fix-missing && \
+  apt-get install -y --no-install-recommends yarn nodejs && \
+  apt-get clean && \
+  rm -rf /var/lib/apt/lists/*
```

Comment:
Installing yarn and nodejs within the docker image, from the official repositories.